docker-compose.yml

```diff
@@ -29,7 +29,7 @@ services:
     container_name: "gost-localstack_main"
     image: localstack/localstack
     ports:
-      - 4566:4566           # LocalStack Gateway
+      - 4566:4566           # SQS
```

Comment:
Small nit, but I recommend reverting this change in the spirit of maintaining accuracy and future clarity. Port 4566 is LocalStack's gateway port through which most (all?) service requests are delivered, and is not exclusive to SQS.